Naivart/Controllers/KingdomController.cs

```diff
@@ -213,6 +214,26 @@ public IActionResult UpgradeTroops([FromRoute] long id, [FromBody] UpgradeTroops
             var upgradeTroopsResponse = new UpgradeTroopsResponse() { status = "ok" };
             return Ok(upgradeTroopsResponse);
         }
+        
+        [Authorize]
+        [HttpPost("kingdoms/{id}/battles")]
+        public IActionResult Battles([FromRoute] long id, [FromBody] BattleTargetRequest input)
+        {
+            var model = KingdomService
+                                       .Battle(input, id, HttpContext.User.Identity.Name, out int status, out string result);
```

Comment:
pls :)

Reply:
v kodu to je primo pod tim KingdomService, dal jsem to dolu aby bylo splneno pocet radku